manifests/repo.pp

```diff
@@ -22,4 +22,14 @@
     enabled  => true,
   }
   -> anchor { 'katello::repo': } # lint:ignore:anchor_resource
+
+  if $facts['os']['release']['major'] == '8' and ($repo_version == 'nightly' or versioncmp($repo_version, '4.3') >= 0) {
```

Comment:
Should this be on non-EL7 in preparation for EL9?
```suggestion
  if $facts['os']['release']['major'] != '9' and ($repo_version == 'nightly' or versioncmp($repo_version, '4.3') >= 0) {
```

Reply:
you surely meant `!= 7` and yeah, probably, but I copied this from the `puppet-foreman` module, and that also had `== 8`.

Reply:
I indeed mean that and it only occurred to me when reviewing this patch.